docs/userGuide/contentAuthoring.md

```diff
@@ -137,10 +160,13 @@ MarkBind usage:
 <code>{<span></span>{glyphicon_hand_right}}</code>
 </tip-box>
 
-Use these [available glyphs](https://getbootstrap.com/docs/3.3/components/#glyphicons) on your site by surrounding a glyph's name with double curly braces.  
-e.g. <code>{<span></span>{glyphicon_hand_right}}</code> will be rendered as <code><span class='glyphicon glyphicon-hand-right' aria-hidden='true'></span></code>
+Here is the [list of provided glyphs](https://getbootstrap.com/docs/3.3/components/#glyphicons) from Bootstrap.
+
+Note that you will have to modify the glyph's name first before being able to use it within MarkBind.<br>It is done by replacing all instances of `-` with `_` in the glyph's name.

```

Comment:
The next line is an example, but this statement does not have a `<br>` at the end, unlike the others.